android/samples/westerosemployees-hybrid/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.liferay.mobile.screens.westerosemployees_hybrid">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
pretty sure this is not formatted. The idea configuration is uploaded here, import it https://github.com/liferay/liferay-screens/blob/develop/android/android-studio-inspections.xml and auto format